tests/conftest.py

```diff
@@ -37,3 +33,22 @@ def generate_expense(db, generate_house):
     return Expenses.create_expense(
         house_name=generate_house, amount=EXPENSE_AMOUNT, date=EXPENSE_DATE, category=EXPENSE_CATEGORY
     )
+
+
+@pytest.fixture
+def request_factory():
+    return RequestFactory()
+
+
+@pytest.fixture
+def generate_get_request(request_factory):
+    get_request = request_factory.get('login/')
+    return get_request
+
```

Comment:
Both request_factory() and generate_get_request() are unused throughout the project and can be deleted

Reply:
@ronyyosef